va/va.h

```diff
@@ -354,6 +364,43 @@ typedef enum
      */
     VAEntrypointEncSliceLP 	= 8,
     VAEntrypointVideoProc       = 10,   /**< Video pre/post-processing. */
+    /**
+     * \brief VAEntrypointFEI
+     *
+     * The purpose of FEI (Flexible Encoding Infrastructure) is to allow applications to
+     * have more controls and trade off quality for speed with their own IPs. A pre-processing
+     * function for getting some statistics and motion vectors is added
+     * and some extra controls for Encode pipeline are provided.
+     * The application can optionally call the statistics function
+     * to get motion vectors and statistics before calling encode function.
```

Comment:
These few lines are for VAEntypointStats, putting them under the description of "VAEntrypointFEI" is confusing.
IMHO, we should move these lines to the comment session of "VAEntypointStats"